metrics/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from topics.models import Article, Topic
+
+
+class Metric(models.Model):
+    article = models.ForeignKey(Article)
+    topic = models.ForeignKey(Topic)
```

Comment:
Hmm I'm really not sure whether this is desirable because then we're splitting the comments left on the same exact article across multiple topics even though the content is the same. I don't see how users would need to refer to a given topic within their comment?